### ISSUE TITLE
loss

### ISSUE THREAD

**Issue by zq130320339** [xw-hu/SINet#9]
I0602 17:23:49.450924 25232 solver.cpp:236] Iteration 450, loss = 2.99931e-05
I0602 17:23:49.450950 25232 solver.cpp:252]     Train net output #0: accuracy_1_5x5 = 1
I0602 17:23:49.450955 25232 solver.cpp:252]     Train net output #1: accuracy_1_5x5 = -1
I0602 17:23:49.450958 25232 solver.cpp:252]     Train net output #2: accuracy_1_7x7 = 1
I0602 17:23:49.450960 25232 solver.cpp:252]     Train net output #3: accuracy_1_7x7 = -1
I0602 17:23:49.450963 25232 solver.cpp:252]     Train net output #4: accuracy_2_5x5 = 1
I0602 17:23:49.450966 25232 solver.cpp:252]     Train net output #5: accuracy_2_5x5 = -1
I0602 17:23:49.450968 25232 solver.cpp:252]     Train net output #6: accuracy_2_7x7 = 1
I0602 17:23:49.450970 25232 solver.cpp:252]     Train net output #7: accuracy_2_7x7 = -1
I0602 17:23:49.450973 25232 solver.cpp:252]     Train net output #8: accuracy_3_5x5 = 1
I0602 17:23:49.450976 25232 solver.cpp:252]     Train net output #9: accuracy_3_5x5 = -1
I0602 17:23:49.450978 25232 solver.cpp:252]     Train net output #10: accuracy_3_7x7 = 1
I0602 17:23:49.450981 25232 solver.cpp:252]     Train net output #11: accuracy_3_7x7 = -1
I0602 17:23:49.450984 25232 solver.cpp:252]     Train net output #12: accuracy_4_5x5 = 1
I0602 17:23:49.450986 25232 solver.cpp:252]     Train net output #13: accuracy_4_5x5 = -1
I0602 17:23:49.450989 25232 solver.cpp:252]     Train net output #14: accuracy_4_7x7 = 1
I0602 17:23:49.450992 25232 solver.cpp:252]     Train net output #15: accuracy_4_7x7 = -1
I0602 17:23:49.450994 25232 solver.cpp:252]     Train net output #16: boxiou_1_5x5 = -1
I0602 17:23:49.450997 25232 solver.cpp:252]     Train net output #17: boxiou_1_7x7 = -1
I0602 17:23:49.450999 25232 solver.cpp:252]     Train net output #18: boxiou_2_5x5 = -1
I0602 17:23:49.451016 25232 solver.cpp:252]     Train net output #19: boxiou_2_7x7 = -1
I0602 17:23:49.451020 25232 solver.cpp:252]     Train net output #20: boxiou_3_5x5 = -1
I0602 17:23:49.451022 25232 solver.cpp:252]     Train net output #21: boxiou_3_7x7 = -1
I0602 17:23:49.451025 25232 solver.cpp:252]     Train net output #22: boxiou_4_5x5 = -1
I0602 17:23:49.451027 25232 solver.cpp:252]     Train net output #23: boxiou_4_7x7 = -1
I0602 17:23:49.451033 25232 solver.cpp:252]     Train net output #24: loss_1_5x5 = 0 (* 0.9 = 0 loss)
I0602 17:23:49.451038 25232 solver.cpp:252]     Train net output #25: loss_1_5x5 = 0 (* 0.9 = 0 loss)
I0602 17:23:49.451041 25232 solver.cpp:252]     Train net output #26: loss_1_7x7 = 0 (* 0.9 = 0 loss)
I0602 17:23:49.451045 25232 solver.cpp:252]     Train net output #27: loss_1_7x7 = 0 (* 0.9 = 0 loss)
I0602 17:23:49.451050 25232 solver.cpp:252]     Train net output #28: loss_2_5x5 = 1.71173e-07 (* 1 = 1.71173e-07 loss)
I0602 17:23:49.451053 25232 solver.cpp:252]     Train net output #29: loss_2_5x5 = 0 (* 1 = 0 loss)
I0602 17:23:49.451056 25232 solver.cpp:252]     Train net output #30: loss_2_7x7 = 5.29819e-08 (* 1 = 5.29819e-08 loss)
I0602 17:23:49.451061 25232 solver.cpp:252]     Train net output #31: loss_2_7x7 = 0 (* 1 = 0 loss)
I0602 17:23:49.451064 25232 solver.cpp:252]     Train net output #32: loss_3_5x5 = 0.000294763 (* 1 = 0.000294763 loss)
I0602 17:23:49.451067 25232 solver.cpp:252]     Train net output #33: loss_3_5x5 = 0 (* 1 = 0 loss)
I0602 17:23:49.451071 25232 solver.cpp:252]     Train net output #34: loss_3_7x7 = 5.50197e-08 (* 1 = 5.50197e-08 loss)
I0602 17:23:49.451074 25232 solver.cpp:252]     Train net output #35: loss_3_7x7 = 0 (* 1 = 0 loss)
I0602 17:23:49.451079 25232 solver.cpp:252]     Train net output #36: loss_4_5x5 = 1.95372e-06 (* 1 = 1.95372e-06 loss)
I0602 17:23:49.451082 25232 solver.cpp:252]     Train net output #37: loss_4_5x5 = 0 (* 1 = 0 loss)
I0602 17:23:49.451086 25232 solver.cpp:252]     Train net output #38: loss_4_7x7 = 3.18513e-06 (* 1 = 3.18513e-06 loss)
I0602 17:23:49.451089 25232 solver.cpp:252]     Train net output #39: loss_4_7x7 = 0 (* 1 = 0 loss)
I0602 17:23:49.451093 25232 sgd_solver.cpp:106] Iteration 450, lr = 5e-05

您好,我想问一下这个损失和正确率是正常的吗

**Comment by zq130320339** [comment]
I0603 11:06:01.388480  7417 solver.cpp:236] Iteration 1100, loss = 0.00270681
I0603 11:06:01.388509  7417 solver.cpp:252]     Train net output #0: accuracy_1_5x5 = 1
I0603 11:06:01.388514  7417 solver.cpp:252]     Train net output #1: accuracy_1_5x5 = -1
I0603 11:06:01.388516  7417 solver.cpp:252]     Train net output #2: accuracy_1_7x7 = 1
I0603 11:06:01.388519  7417 solver.cpp:252]     Train net output #3: accuracy_1_7x7 = -1
I0603 11:06:01.388521  7417 solver.cpp:252]     Train net output #4: accuracy_2_5x5 = 1
I0603 11:06:01.388540  7417 solver.cpp:252]     Train net output #5: accuracy_2_5x5 = -1
I0603 11:06:01.388542  7417 solver.cpp:252]     Train net output #6: accuracy_2_7x7 = 1
I0603 11:06:01.388545  7417 solver.cpp:252]     Train net output #7: accuracy_2_7x7 = -1
I0603 11:06:01.388548  7417 solver.cpp:252]     Train net output #8: accuracy_3_5x5 = 1
I0603 11:06:01.388551  7417 solver.cpp:252]     Train net output #9: accuracy_3_5x5 = -1
I0603 11:06:01.388555  7417 solver.cpp:252]     Train net output #10: accuracy_3_7x7 = 1
I0603 11:06:01.388557  7417 solver.cpp:252]     Train net output #11: accuracy_3_7x7 = -1
I0603 11:06:01.388559  7417 solver.cpp:252]     Train net output #12: accuracy_4_5x5 = 1
I0603 11:06:01.388563  7417 solver.cpp:252]     Train net output #13: accuracy_4_5x5 = -1
I0603 11:06:01.388573  7417 solver.cpp:252]     Train net output #14: accuracy_4_7x7 = 1
I0603 11:06:01.388579  7417 solver.cpp:252]     Train net output #15: accuracy_4_7x7 = -1
I0603 11:06:01.388583  7417 solver.cpp:252]     Train net output #16: boxiou_1_5x5 = -1
I0603 11:06:01.388588  7417 solver.cpp:252]     Train net output #17: boxiou_1_7x7 = -1
I0603 11:06:01.388592  7417 solver.cpp:252]     Train net output #18: boxiou_2_5x5 = -1
I0603 11:06:01.388597  7417 solver.cpp:252]     Train net output #19: boxiou_2_7x7 = -1
I0603 11:06:01.388602  7417 solver.cpp:252]     Train net output #20: boxiou_3_5x5 = -1
I0603 11:06:01.388607  7417 solver.cpp:252]     Train net output #21: boxiou_3_7x7 = -1
I0603 11:06:01.388610  7417 solver.cpp:252]     Train net output #22: boxiou_4_5x5 = -1
I0603 11:06:01.388614  7417 solver.cpp:252]     Train net output #23: boxiou_4_7x7 = -1
I0603 11:06:01.388618  7417 solver.cpp:252]     Train net output #24: cls_accuracy_large = 1
I0603 11:06:01.388623  7417 solver.cpp:252]     Train net output #25: cls_accuracy_small = 1
I0603 11:06:01.388631  7417 solver.cpp:252]     Train net output #26: loss_1_5x5 = 0 (* 0.9 = 0 loss)
I0603 11:06:01.388640  7417 solver.cpp:252]     Train net output #27: loss_1_5x5 = 0 (* 0.9 = 0 loss)
I0603 11:06:01.388648  7417 solver.cpp:252]     Train net output #28: loss_1_7x7 = 0 (* 0.9 = 0 loss)
I0603 11:06:01.388656  7417 solver.cpp:252]     Train net output #29: loss_1_7x7 = 0 (* 0.9 = 0 loss)
I0603 11:06:01.388662  7417 solver.cpp:252]     Train net output #30: loss_2_5x5 = 1.45701e-06 (* 1 = 1.45701e-06 loss)
I0603 11:06:01.388671  7417 solver.cpp:252]     Train net output #31: loss_2_5x5 = 0 (* 1 = 0 loss)
I0603 11:06:01.388679  7417 solver.cpp:252]     Train net output #32: loss_2_7x7 = 1.85437e-07 (* 1 = 1.85437e-07 loss)
I0603 11:06:01.388687  7417 solver.cpp:252]     Train net output #33: loss_2_7x7 = 0 (* 1 = 0 loss)
I0603 11:06:01.388696  7417 solver.cpp:252]     Train net output #34: loss_3_5x5 = 5.08627e-07 (* 1 = 5.08627e-07 loss)
I0603 11:06:01.388703  7417 solver.cpp:252]     Train net output #35: loss_3_5x5 = 0 (* 1 = 0 loss)
I0603 11:06:01.388712  7417 solver.cpp:252]     Train net output #36: loss_3_7x7 = 1.58946e-08 (* 1 = 1.58946e-08 loss)
I0603 11:06:01.388720  7417 solver.cpp:252]     Train net output #37: loss_3_7x7 = 0 (* 1 = 0 loss)
I0603 11:06:01.388728  7417 solver.cpp:252]     Train net output #38: loss_4_5x5 = 0 (* 1 = 0 loss)
I0603 11:06:01.388736  7417 solver.cpp:252]     Train net output #39: loss_4_5x5 = 0 (* 1 = 0 loss)
I0603 11:06:01.388744  7417 solver.cpp:252]     Train net output #40: loss_4_7x7 = 0 (* 1 = 0 loss)
I0603 11:06:01.388751  7417 solver.cpp:252]     Train net output #41: loss_4_7x7 = 0 (* 1 = 0 loss)
I0603 11:06:01.388759  7417 solver.cpp:252]     Train net output #42: loss_bbox_large = 0 (* 1 = 0 loss)
I0603 11:06:01.388767  7417 solver.cpp:252]     Train net output #43: loss_bbox_small = 0 (* 1 = 0 loss)
I0603 11:06:01.388775  7417 solver.cpp:252]     Train net output #44: loss_cls_large = 0.00038265 (* 1 = 0.00038265 loss)
I0603 11:06:01.388784  7417 solver.cpp:252]     Train net output #45: loss_cls_small = 0.000599586 (* 1 = 0.000599586 loss)
I0603 11:06:01.388792  7417 sgd_solver.cpp:106] Iteration 1100, lr = 0.0001
正确率一直是1,感觉有点不正常,问题可能出现在哪里?
